package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "coc-todolist",
-  "version": "1.1.1",
```

Comment:
Thanks for your contribution! 
Here could you please stay the version number as the old one? Because I am using an auto-publish script to handle this.

Reply:
Sure, reverted.